MidtransCoreKit.podspec

```diff
@@ -7,7 +7,9 @@ s.homepage         = "https://veritrans.co.id/"
 s.license          = 'MIT'
 s.author           = { "veritrans" => "dev@veritrans.co.id" }
 s.source           = { :git => 'https://github.com/veritrans/Veritrans-ios-sdk.git', :tag => s.version}
-s.platform     = :ios, '9.0'
+s.platform     = :ios, '12.0'
```

Comment:
whats the reason to update  `platform/ios.deployment_target` to 12?

Reply:
hmm i think I forgot to revert this commit on the podspec, will revert this.